app-text/cmark/cmark-0.30.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+
+inherit cmake python-any-r1
+
+DESCRIPTION="CommonMark parsing and rendering library and program in C"
+HOMEPAGE="https://github.com/commonmark/cmark"
+SRC_URI="https://github.com/commonmark/cmark/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0/0.30.0"
```

Comment:
shouldn't this be bumped?
```
 * QA: comparing =app-text/cmark-0.30.0 with app-text/cmark-0.30.1/image
 * SONAME:-libcmark.so.0.30.0(64)
 * SONAME:+libcmark.so.0.30.1(64)
```
Maybe a good occasion to just make this `SLOT="0/${PV}"` if it's like that every time rather than risk forgetting.